external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDelegationTokenSuite.scala

```diff
@@ -62,7 +62,7 @@ class KafkaDelegationTokenSuite extends StreamTest with SharedSparkSession with
     }
   }
 
-  test("Roundtrip") {
+  testRetry("Roundtrip", 3) {
```

Comment:
@zsxwing @gaborgsomogyi Should we disable this flaky test first and create a blocker ticket for 3.0? 

If we keep hitting the test failure, it will impact the productivity of the open source development.

Reply:
Sounds fine to me.

Reply:
+1

Reply:
Late +1